src/print.jl

```diff
@@ -436,15 +443,15 @@ function cont_str(mode, j, sym::PrintSymbols)
         end
     end
     num_dims = length(data.indexsets)
-    idxvars = Array{String}(num_dims)
+    idxvars = Array{String}(undef, num_dims)
     dimidx = 1
     for i in 1:num_dims
         if data.indexexprs[i].idxvar == nothing
             while DIMS[dimidx] in locvars
                 dimidx += 1
             end
             if dimidx > length(DIMS)
-                error("Unexpectedly ran out of indices")
+                throw(ErrorException("Unexpectedly ran out of indices"))
```

Comment:
I'd prefer `Compat.@error` here.

Reply:
That doesn't actually throw an exception.  At least it shouldn't, because `@error` in 0.7 doesn't throw an exception.

Reply:
(Note that `error` in 0.6 *did* throw an exception.)

Reply:
Wait, `error` is not deprecated in 0.7. Why does this need to change?

Reply:
oh wow... not sure why I thought it was then.  Ok changing...